README.md

```diff
@@ -27,6 +27,7 @@ This crate is fully compatible with Cargo. Just add it to your `Cargo.toml`:
 ```toml
 [dependencies]
 docopt = "0.8"
+serde = "1.0"
 serde_derive = "1.0" # if you're using `derive(Deserialize)`
```

Comment:
Could you add the `# if you're using derive(Deserialize)` comment to the `serde` line as well please?